app/assets/stylesheets/application.css

```diff
@@ -1,18 +0,0 @@
-/*
```

Comment:
You can remove this file

Reply:
> You can remove this file

this file is empty ^ I dont have idea why do you see this